bundles/org.openhab.binding.lcn/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -640,24 +640,25 @@
 
 	<channel-type id="remotecontrolkeys">
 		<kind>trigger</kind>
-		<label>Remote Control Keys</label>
+		<label>Remote Control (Keys)</label>
 		<event/>
 	</channel-type>
 
 	<channel-type id="remotecontrolcodes">
 		<kind>trigger</kind>
-		<label>Remote Control with Access Control Code</label>
+		<label>Remote Control (Codes)</label>
 		<event/>
 	</channel-type>
 
 	<channel-type id="remotecontrolsbatterylow">
 		<kind>trigger</kind>
-		<label>Remote Control Low Battery</label>
+		<label>Low Battery</label>
 		<event/>
 	</channel-type>
 
 	<channel-group-type id="hostcommands">
-		<label>Host Command (to openHAB)</label>
+		<label>Host Command</label>
```

Comment:
```suggestion
		<label>Host Commands</label>
```
Just my OCD wanting to keep the label in line with the channel ID

Reply:
I would claim that the id is actually chosen badly, but since the other ids and labels are named in the same way, I agree to better change the label here as well.